app/views/layouts/markdown.html.erb

```diff
@@ -3,7 +3,7 @@
 <% end %>
 
 <% content_for :content do %>
-  <div class="container-fluid">
+  <div class="container">
```

Comment:
Switching to the fluid-less class may have implications for responsiveness. Did you test the layout at various screen sizes, from phone all the way up to full screen?

Reply:
I dragged the browser window width from the narrowest (mobile width) to the widest (two monitors wide) and everything looks the way I'd expect.